sessions/slides/introduction-to-biases-in-epidemiological-delays.qmd

```diff
@@ -0,0 +1,93 @@
+---
+title: "Introduction to biases in epidemiological delays"
+author: "Nowcasting and forecasting of infectious disease dynamics"
+engine: knitr
+format:
+  revealjs:
+    output: slides/introduction-to-biases-in-epidemiological-delays.html
+    footer: "Introduction to biases in epidemiological delays"
+---
+
+```{r seed}
+set.seed(123)
+```
+# Biases in epidemiological delays
+
+Why might our estimates of epidemiological delays be biased?
+
+. . .
+
+::: {.fragment .fade-out}
+- data reliability and representativeness
+:::
+- intrinsic issues with data collection and recording
+
+## Issue #1: Double censoring
+
+- reporting of events usually as a **date** (not date + precise time)
+- for short delays this can make quite a difference
+
+## Double censoring: example {.smaller}
+
+We are trying to estimate an incubation period. For person A we know exposure happened on day 1 and symptom onset on day 3.
+
+![](figures/censoring_1.png)
+
+## Double censoring: example {.smaller}
+
+We are trying to estimate an incubation period. For person A we know exposure happened on day 1 and symptom onset on day 3.
+
+![](figures/censoring_2.png)
+
+## Double censoring: example {.smaller}
+
+We are trying to estimate an incubation period. For person A we know exposure happened on day 1 and symptom onset on day 3.
+
+![](figures/censoring_final.png)
+
+## Double censoring: example {.smaller}
+
+We are trying to estimate an incubation period. For person A we know exposure happened on day 1 and symptom onset on day 3.
+
+![](figures/censoring_final.png)
+
+The *true* incubation period of A could be anywhere between 1 and 3 days (but not all equally likely).
+
+## Issue #2: right truncation
+
+- reporting of events can be triggered by the **second** event
+- in that case, longer delays might be missing because they **have not occurred yet**
```

Comment:
```suggestion
- in that case, longer delays might be missing because whilst the primary events have occurred the secondary events **have not occurred yet**
```